### PR TITLE
fix: scaleway api endpoint

### PR DESCRIPTION
**Pull request by abarbare**
##### SUMMARY
Update Scaleway API endpoint

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
scaleway_compute
scaleway_ip
scaleway_security_group
scaleway_security_group_rule
scaleway_snapshot 
scaleway_volume

### CODE REVIEW

File: lib/ansible/module_utils/scaleway.py [hunk 1]
```diff
@@ -161,14 +161,14 @@ def warn(self, x):
 
 
 SCALEWAY_LOCATION = {
-    'par1': {'name': 'Paris 1', 'country': 'FR', "api_endpoint": 'https://cp-par1.scaleway.com'},
-    'EMEA-FR-PAR1': {'name': 'Paris 1', 'country': 'FR', "api_endpoint": 'https://cp-par1.scaleway.com'},
+    'par1': {'name': 'Paris 1', 'country': 'FR', "api_endpoint": 'https://api.scaleway.com/instance/v1/zones/fr-par-1/'},
```
Comment (kindermoumoute) [inline lib/ansible/module_utils/scaleway.py]: Previous URLs didn't end with a slash, this might create double slash URLs.

**Review by kindermoumoute — APPROVED** [review]
LGTM

**Review by QuentinBrosse — APPROVED** [review]
LGTM

**Comment by gundalow** [comment]
@abarbare @kindermoumoute @QuentinBrosse Thank you for this fix and your reviews.

Merged into `devel` for release in Ansible 2.10.

If you'd like to see this fixed in Ansible 2.9.1 please raise a backport PR including `changelog/fragment` see https://docs.ansible.com/ansible/latest/community/development_process.html#making-your-pr-merge-worthy for more information

**Pull request merged**